src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -67,10 +67,10 @@ export default function SettingsLMSTab({
       .then((response) => {
         setShowNewConfigButtons(true);
         setConfigsLoading(false);
+        // Save all existing configs
+        setExistingConfigsData(camelCaseDictArray(response.data));
         // If the enterprise has existing configs
         if (response.data.length !== 0) {
-          // Save all existing configs
-          setExistingConfigsData(camelCaseDictArray(response.data));
```

Comment:
This line needed to be moved outside of the data length check, otherwise deleting the last remaining config in the list would not update the page.